Content.Server/Tools/Components/WelderComponent.cs

```diff
@@ -103,17 +109,19 @@ protected override void Initialize()
 
             _welderSystem = _entitySystemManager.GetEntitySystem<WelderSystem>();
 
-            Owner.TryGetComponent(out _solutionComponent);
+            Owner.EnsureComponent<SolutionContainerManager>();
             Owner.TryGetComponent(out _spriteComponent);
             Owner.TryGetComponent(out _pointLightComponent);
+            EntitySystem.Get<SolutionContainerSystem>().EnsureSolution(Owner, "welder");
```

Comment:
Like I said elsewhere EnsureSolution should just EnsureComponent on the container manager.